build.rs

```diff
@@ -76,6 +76,11 @@ fn main() {
     }
   }
 
+  // Enable snapshots for x64 builds
+  if env::var("CARGO_CFG_TARGET_ARCH").unwrap() == "x86_64" {
+    println!("cargo:rustc-cfg=feature=\"use-snapshots\"");
```

Comment:
This `use-snapshots` cfg is a bit confusing to me because there's also a GN option called `v8_use_snapshot` which is not effected by this.
Could you add a comment here to this effect that these are unrelated? Maybe rename `use-snapshots` to `load-snapshots` ?